CHANGELOG.md

```diff
@@ -1,9 +1,9 @@
 # Upcoming
 
 * BREAKING CHANGE: Seperate the code from the angular_token_auth into seperate files.
-* Add file linting.
```

Comment:
Well not necessarily, since that's a dev ops change, but `lint the files` is a source change, even if the effects are null